trade-finance-manager-ui/styles/block-header.scss

```diff
@@ -0,0 +1,11 @@
+.block-header {
+  background-color: $govuk-brand-colour;
+  padding: 10px;
+  color: govuk-colour("white");
+  margin-bottom: 40px;
+
+  p {
+    margin-top: 10px;
+    margin-bottom: 10px;
+  }
```

Comment:
Can we use the GOV class names in nunjucks, `govuk-!-margin-bottom-X` etc instead of in the styles?

Reply:
Note: this has now moved to `fee-record-summary.scss` as a result of the changes for the above comment

This is to achieve spacing between the rows in the blue banner, so in retrospect it would be better if we apply the `govuk-body` class to the paragraph elements and apply the margin to the rows themselves.

We could achieve the row margin in nunjucks using `govuk-!-margin-` or I've just found that we can do the following in the scss file:

```
.govuk-grid-row {
  margin-top: govuk-spacing(3);
}

.govuk-grid-row:last-child {
  margin-bottom: govuk-spacing(3);
}
```

Do you have a preference?

Reply:
Thanks @sw-mattw I think we should do it in nunjucks beacuse the majority of the codebase does this vs `govuk-spacing` in scsss, can be consistent

Reply:
Sounds good thanks, updated to use `govuk-!-margin-` in the template